Update vue example app

pekura opened the pull request:
Vue app is now a minimal setup example for a Luigi app with vue.

alexandra-simeonova commented on core/examples/luigi-sample-vue/README.md:
```diff
@@ -4,46 +4,24 @@
-    # The `lerna bootstrap` command executes the Node Package Manager (NPM) installation and links cross-dependencies.
-
-    lerna bootstrap
+    npm install
```
Maybe we can mention in what directory the commands have to be run just to avoid confusion? (Ignore this if it's too obvious 😅)

pekura replied:
:-) They should be executed in the directory where this README.md is. I would say that is clear.

alexandra-simeonova commented on core/examples/luigi-sample-vue/README.md:
````diff
@@ -52,4 +30,4 @@ npm run build
 ```
 
 The build compiles and minifies the source files for production usage.
````
```suggestion
The build compiles and minimizes the source files for production usage.
```

pekura commented:
> Good job 👍
> 
> I thought about 2 issues here, although I'm not sure if they are needed.
> 
> * do we need to mention the vue example in docu https://github.com/SAP/luigi/blob/master/core/examples/README.md ?
> * do we want to adjust this example so it renders well on mobile devices (https://github.com/SAP/luigi/blob/master/docs/luigi-ux-features.md#responsive-application-setup)

Thank you, Maryna!
I added the link to the Vue sample app.
I added the meta viewport as well.

marynaKhromova commented:
> Thank you, Maryna!
> I added the link to the Vue sample app.
> I added the meta viewport as well.
Great, thanks! I would recommend as well to use `responsiveNavigation: 'simpleMobileOnly'` to have burger navigation on mobile. Otherwise, the left navigation takes now almost the whole screen width on mobile resolution.

pekura commented:
> Great, thanks! I would recommend as well to use responsiveNavigation: 'simpleMobileOnly' to have burger navigation on mobile. Otherwise, the left navigation takes now almost the whole screen width on mobile resolution.

Done!

pekura commented:
> Well done :)

Well reviewed ;-)
